src/main/java/dwbh/api/fetchers/VotingFetcherUtils.java

```diff
@@ -65,12 +66,15 @@ private VotingFetcherUtils() {
     UUID votingId = environment.getArgument("votingId");
     String comment = environment.getArgument("comment");
     Integer score = environment.getArgument("score");
+    Boolean anonymous =
```

Comment:
You can change all `Boolean` properties in input java class to `boolean`. In this case you'd only have to write

```groovy
boolean anonymous = environment.getArgument("anonymous");
```
As long as it is a primitive type, it has a default value of false, so you don't have to check for null value.

Reply:
Mmm... I've tried this, and I get a NullPointerException if the argument is not present.

I've changed it to boolean, and set a ternary operator:

boolean anonymous =
        environment.getArgument("anonymous") != null ? environment.getArgument("anonymous") : false;

Reply:
In fact:
```
boolean anonymous =
        environment.getArgument("anonymous") != null
            && (boolean) environment.getArgument("anonymous");
```
Because PMD "Simplified Ternary" rule...